### PR TITLE
Feature/6921

### PR DESCRIPTION
**Pull request by karthik-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6921
*Description of changes:*
Hi. As part of 6921. the idea is to intercept user with a duplicate account warning screen when the user already has an existing account and try to create another account. 
Therefore, when the user clicks create account, 

- In the sbc-common:(sbc-header) component, we capture the redirect url link ( as a property ) and direct to setup-account of sbc-auth. 
- In the sbc-auth, when the user decides not to create account and tries to access existing account, we route to the redirect url that we captured earlier.
- A default url is also used incase there is no redirect url. ( sbc-auth home)
-  we have created a dependency ticket as well https://github.com/bcgov/entity/issues/7757

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    private orgsOfUser: OrgWithAddress[] = []
+    private isLoading: boolean = false
+    @Prop({ default: '' }) redirectToUrl !: string
+
+    private async mounted () {
+      try {
+        this.isLoading = true
+        await this.addOrgSettings(this.currentOrganization)
+        const userSettings: UserSettings[] = await this.getUserSettings(this.userProfile?.keycloakGuid)
+        for (let i = 0; i < userSettings.length && userSettings.length > 0; i++) {
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue:80]: array.map will be easy  ?

File: auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        }
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.log(`Error while loading duplicate accounts ${err}`)
+      } finally {
+        this.isLoading = false
+      }
+    }
+
+    private navigateToRedirectUrl (accountId: number): void {
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue:98]: account id not used

File: auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                      <strong>{{ org.name && org.name.slice(0,1) && org.name.slice(0,1).toUpperCase() }}</strong>
+                  </v-avatar>
+                </div>
+                <div class="text-left ml-2">
+                <h4 class="font-weight-bold ">{{ org.name }}</h4>
+                <p class="mb-0">{{ org.addressLine }}</p>
+              </div>
+            </div>
+            <div class="text-right">
+              <v-btn large color="primary" @click="navigateToRedirectUrl(org.id)" title="Go to Business Dashboard" data-test="goto-dashboard-button">Access Account</v-btn>
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue:34]: title ?

File: auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop, Vue } from 'vue-property-decorator'
+import { OrgWithAddress, Organization } from '@/models/Organization'
+import { Pages, SessionStorageKeys } from '@/util/constants'
+import { User, UserSettings } from '@/models/user'
+import { Address } from '@/models/address'
+import ConfigHelper from '@/util/config-helper'
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue:56]: not used ?

File: auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue (hunk 1 above)
Comment (karthik-aot) [reply auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue:80]: thanks. refactored

File: auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue (hunk 3 above)
Comment (karthik-aot) [reply auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue]: thanks refactored :)

File: auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue (hunk 2 above)
Comment (karthik-aot) [reply auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue:98]: @saravanpa-aot  we are syncing org and org settings, is this fine ?

File: auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue (hunk 4 above)
Comment (karthik-aot) [reply auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue:56]: Thanks. addressed

**Review by shabeeb-aot — APPROVED** [review]

**Pull request merged**